source/class/qx/tool/cli/Cli.js

```diff
@@ -356,8 +361,7 @@ Version: v${await qx.tool.config.Utils.getQxVersion()}
             (await qx.tool.utils.Json.loadJsonAsync(name)) || lockfileContent;
         } catch (ex) {
           // Nothing
-        }
-        // check semver-type compatibility (i.e. compatible as long as major version stays the same)
+        } // check semver-type compatibility (i.e. compatible as long as major version stays the same)
```

Comment:
This looks like an accidental joining of the two lines. I don't believe the comment belongs after the ending brace, right?

Reply:
Done

Reply:
This was marked as resolved, but doesn't appear to have changed. The comment is still showing as joined onto the line with the closing brace.